### DISCUSSION TITLE
Binding a command to a lone modifier key

### DISCUSSION THREAD

**Discussion by m-col** [qtile/qtile#2780] [Ideas]
Following on from https://github.com/qtile/qtile/issues/2778

Currently one can use a modifier in a keybinding e.g. `Key([], "Super_L", lazy.spawn("rofi -show"))` but the keybinding is executed immediately upon modifier press. This means any keybindings that depend on that modifier _as a modifier_ always trigger this binding first which is a bit weird. It would be nice if this was handled properly e.g. waiting for a key release event when a lone modifier is used as a binding and instead executing on that. Similarly, Super_L+Super_R might be nice - not sure how this would work! Use one as a modifier? Who knows.

**Comment by CSpencerND** [comment]
I personally would like to be able to use super by itself. I can't do it with sxhkd either. At least not that I know of.
